unitTests/controllers/FacetQueryControllerTest.ts

```diff
@@ -131,6 +141,18 @@ export function FacetQueryControllerTest() {
           expect(groupByRequest.constantQueryOverride).toBe('an additional filter');
           expect(groupByRequest.advancedQueryOverride).toBe('@uri');
         });
+
+        it("should add a basic query override if the facet is configured with an additional filter and there's a basic expression in the query", () => {
```

Comment:
Hmm, do we need to repeat the test when one value is selected/excluded, or is just the test on line 95 enough?

Reply:
It's helpful, yes, because query override can act differently when values are excluded/selected.